compiler-rt/lib/radsan/tests/CMakeLists.txt

```diff
@@ -54,8 +53,6 @@ if(APPLE)
 endif()
 
 foreach(arch ${RADSAN_TEST_ARCH})
-  set(arch ${COMPILER_RT_DEFAULT_TARGET_ARCH})
```

Comment:
This is the key difference, the rest of the changes are just removing dead configuration code!